lib/controller/python/controller/mouse.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from controller.wb import wb
+import ctypes
+import struct
+from typing import Union
+
+
+class MouseState():
+    def __init__(self, u: float, v: float, x: float, y: float, z: float, left: bool, middle: bool, right: bool):
+        self.u = u
+        self.v = v
+        self.x = x
+        self.y = y
+        self.z = z
+        self.left = left
+        self.middle = middle
+        self.right = right
+
+
+class Mouse():
+    wb.wb_mouse_get_state.restype = ctypes.POINTER(ctypes.c_ubyte)
+
+    def __init__(self, sampling_period: int = None):
+        if sampling_period != 0:
+            self.sampling_period = int(wb.wb_robot_get_basic_time_step()) if sampling_period is None else sampling_period
+
+    def enable(self, p: int):
+        wb.wb_mouse_enable(p)
+
+    def disable(self):
+        wb.wb_mouse_disable()
+
+    def getSamplingPeriod(self) -> int:
+        return self.sampling_period
+
+    def getState(self) -> MouseState:
```

Comment:
It causes the controller to crash.
I used this code in the run function of a device example:
```
mouse = Mouse()
mouse.enable(self.timeStep)
while self.step(self.timeStep) != -1:
    print(mouse.getState())
```